MdeModulePkg/Core/Dxe/DxeMain.h

```diff
@@ -1580,6 +1581,27 @@ CoreWaitForEvent (
 
 // MU_CHANGE begin
 
+/**
+  Stops execution until an event is signaled.
+
+  @param  NumberOfEvents         The number of events in the UserEvents array
+  @param  UserEvents             An array of EFI_EVENT
+  @param  UserIndex              Pointer to the index of the event which
+                                 satisfied the wait condition

```

Comment:
We've been asking for param doc updates in other PRs, so might as well add them here.

```suggestion
  @param[in]  NumberOfEvents     The number of events in the UserEvents array
  @param[in]  UserEvents         An array of EFI_EVENT
  @param[out]  UserIndex         Pointer to the index of the event which
                                 satisfied the wait condition
```

Reply:
Added. Thanks!